node_lambnik/src/demo/datatype-demo.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Datatype Demo</title>
+
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css" rel="stylesheet" type="text/css" />
```

Comment:
This is likely of little consequence, especially as a demo, but I'd recommend using the much more recent 1.x series of leaflet.

Reply:
I left it at 0.7.x because I couldn't get 1.0 to work with utf grids and didn't want to waste time on it at the time, but I probably should update that.